test/tests/haproxy-basics/run.sh

```diff
@@ -8,7 +8,7 @@ dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 image="$1"
 
-clientImage='buildpack-deps:jessie-curl'
+clientImage='buildpack-deps:curl'
```

Comment:
This would be good to keep pinned to a particular suite (so if you want to update it, swapping to `stretch-curl` would be A+).

Reply:
Seems I have missed this before: Do you really think `curl` would break? That's why I removed the “pin” to the Debian version altogether: I consider it more likely that the image stops being updated than curl being broken.

Reply:
Honestly, we've seen stranger things -- we do have quite a few tests pinning to a particular suite though, so keeping that consistent is safer.  It's trivially easy to update them later (and we should probably do a pass moving them all to `stretch` sometime soon).